app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md

```diff
@@ -25,11 +25,16 @@ quote:
   q-jane-wilkinson:
     text: "I specifically wanted to work with children identified with Special Educational Needs and Disabilities (SEND). I was fortunate to find a teacher training course provider that was able to allow me to practice in a mainstream school that also hosted a hearing impairment unit. And another school that offered support for children with a variety of learning needs. They were great at teaching me and letting me practice my classroom management skills early on. I was able to research and understand ways to differentiate resources for children who were on the autistic spectrum. And I was able to create and implement a scheme of work to reflect on and evaluate for later use once qualified."
     name: "Jane Wilkinson, former teacher and teacher training adviser"
+  q-gareth-williams:
+    text: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam et semper ligula. Donec ut gravida magna. Curabitur in urna mauris. Nulla accumsan et nibh eu porta."
+    name: "Gareth Williams, Interation Designer"
```

Comment:
"Interaction"

Reply:
we need to delete this altogether before this PR is merged!